src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt

```diff
@@ -38,19 +40,21 @@ class RelationalDatabaseEventStore<M: EventMetadata> @PublishedApi internal cons
         inline fun <reified M: EventMetadata> create(
             synchronousEventProcessors: List<EventProcessor<M>>,
             db: Database,
-            objectMapper: ObjectMapper = defaultObjectMapper
```

Comment:
1.4 trailing commas!

Reply:
I'm not a fan :P

Reply:
Gonna keep the comma style consistent for now